src/vision/status_quo/barbara_compares_some_cpp_code.md

```diff
@@ -91,6 +91,12 @@ would have probably required figuring out how to use `perf` or some similar tool
 
 ## 🤔 Frequently Asked Questions
 
+### **Is this actually the correct solution?**
+* Only in part. It may cause other kinds of contention or blocking on the runtime. The deserialization work probably needs to be wrapped in something like [`block_in_place`](https://docs.rs/tokio/1.5.0/tokio/task/fn.block_in_place.html), so that other tasks are not starved on the runtime, or might want to use [`spawn_blocking`](https://docs.rs/tokio/1.5.0/tokio/task/fn.spawn_blocking.html). There are some important caveats/details that matter:
+  * This is dependent on how the runtime works.
+  * `block_in_place` + `tokio::spawn`/`spawn_blocking` might be better if the caller wants to control concurrency, as spawning is heavyweight when the deserialization work happens to be small.
```

Comment:
Using `block_in_place` and `spawn_blocking` together is unlikely to be helpful - the choices should be between `spawn_blocking` and `block_in_place` + `tokio::spawn`.